getting-started/analyst-exercise/output.xml

```diff
@@ -30,15 +30,42 @@
 	http://stix.mitre.org/common-1 http://stix.mitre.org/XMLSchema/common/1.1.1/stix_common.xsd
 	http://stix.mitre.org/default_vocabularies-1 http://stix.mitre.org/XMLSchema/default_vocabularies/1.1.1/stix_default_vocabularies.xsd
 	http://stix.mitre.org/stix-1 http://stix.mitre.org/XMLSchema/core/1.1.1/stix_core.xsd" id="example:Package-e622beb9-3781-4dd7-9787-ac21ba84d898" version="1.1.1" timestamp="2014-07-25T18:16:44.543625+00:00">
-    <stix:STIX_Header>
-        <stix:Package_Intent xsi:type="stixVocabs:PackageIntentVocab-1.0">Threat Report</stix:Package_Intent>
-        <stix:Description>Indicators for FireEye report on Saffron</stix:Description>
-        <stix:Information_Source>
-            <stixCommon:Time>
-                <cyboxCommon:Produced_Time>2014-05-15T00:00:00</cyboxCommon:Produced_Time>
-            </stixCommon:Time>
-        </stix:Information_Source>
-    </stix:STIX_Header>
+    
+    <stix:Reports>
+        <stix:Report id="fireeye-ajax" xsi:type="report:ReportType">
+            <report:Header>
+                <report:Title> FireEye Ajax/Saffron report</report:Title>
+                <report:Description>Indicators and background on a computer intrusion</report:Description>
+            <report:Information_Source>
+                <stixCommon:Time>
+                    <cyboxCommon:Produced_Time>2014-05-15T00:00:00</cyboxCommon:Produced_Time>
+                </stixCommon:Time>
+            </report:Information_Source>    
+            
+            <report:Package_Intent xsi:type="stixVocabs:PackageIntentVocab-1.0">Threat Report</report:Package_Intent>    
```

Comment:
This should use `ReportIntentVocab-1.0`, not `PackageIntentVocab-1.0` (they are identical). I'll fix and merge, just wanted you to know for the future.